### ISSUE TITLE
HUE 4.7 "'ascii' codec can't decode byte" error

### ISSUE THREAD

**Issue by alant94** [cloudera/hue#1131]
**Is the issue already present in https://github.com/cloudera/hue/issues or discussed in the forum https://discourse.gethue.com?**
No, at least I couldn't find it

**Describe the bug:**
When running Hive or Impala query that contains special characters, for example `çã` the error is returned: 
`'ascii' codec can't decode byte 0xc3 in position 90: ordinal not in range(128)`

Log file with more details attached. [hue_47_ascii_codec_cant_decode_error.txt](https://github.com/cloudera/hue/files/4644647/hue_47_ascii_codec_cant_decode_error.txt)
The same query works without errors when executed from Hue 4.6

**Steps to reproduce it?**
Create table with data that contains symbols `ç` or `ã` in string fields and try to query it like this:
`select * from db.table where name='Don Pelição'`
The query works fine if it doesn't contain special characters.

**Hue version or source? (e.g. open source 4.5, CDH 5.16, CDP 1.0...). System info (e.g. OS, Browser...).**
Open source 4.7. System info - OS: Windows 10; Browsers: Firefox and Chrome.

**Comment by romainr** [comment]
Indeed, this is has not been working for a while. There are a few tricky unicode places to handle correctly. I faced a few of those while porting to Python 3 (in the fetch result call) and will see if possible to fix at some point.

**Comment by alant94** [comment]
@romainr thanks for a quick reply! 
In phrase "has not been working for a while" what amount of time is meant? Just want to highlight that this problem was noticed by my team only in Hue 4.7. So we rolled back to 4.6 where queries with special characters work fine.

**Comment by romainr** [comment]
Indeed, I see the same with the HiveServer2 interface.

I was thinking about another issue, which is actually different:
`SELECT 'fix Lỗi CS1: ngày tháng'`

**Comment by alant94** [comment]
> This issue is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days.

comment

**Comment by milletnis** [comment]
we are facing the same problem. With HUE 4.8.0 some LDAP users are not able to login anymore.
Same users and same LDAP backend but Hue 4.4.0 works fine for us.

```
[18/Mar/2021 21:06:02 +0000] decorators   INFO     AXES: Calling decorated function: dt_login
[18/Mar/2021 21:06:02 +0000] backend      INFO     Augmenting users with class: <class 'desktop.auth.backend.DefaultUserAugmentor'>
[18/Mar/2021 21:06:03 +0000] forms        ERROR    LDAP auth error: 'ascii' codec can't decode byte 0xc3 in position 18: ordinal not in range(128)
[18/Mar/2021 21:06:03 +0000] access       WARNING  127.0.0.1 -anon- - "POST /hue/accounts/login HTTP/1.1" --- Failed login for user: dummy
[18/Mar/2021 21:06:03 +0000] access       WARNING  127.0.0.1 -anon- - "POST /hue/accounts/login HTTP/1.1" --- Failed login for user: dummy
[18/Mar/2021 21:06:03 +0000] decorators   INFO     AXES: New login failure by 127.0.0.1. Creating access record.
```

**Comment by Akhilsnaik** [comment]
Fixed as part of : https://github.com/cloudera/hue/commit/13822608d8302c04bee110e281e1f5a1b20b7fa6

**Comment by Akhilsnaik** [comment]
Closing the issue as it has been fixed as part of https://github.com/cloudera/hue/commit/13822608d8302c04bee110e281e1f5a1b20b7fa6

**Comment by arturbrandysonwelo** [comment]
This isn't been fixed for the LDAP problem in Hue 4.10 as @milletnis was writing.
